File: scaffold/src/main/java/top/xuqingquan/web/agent/IVideo.java
package top.xuqingquan.web.agent;

import android.view.View;

public interface IVideo {

    void onShowCustomView(View view, android.webkit.WebChromeClient.CustomViewCallback callback);

    void onShowCustomView(View view, com.tencent.smtt.export.external.interfaces.IX5WebChromeClient.CustomViewCallback callback);

    void onHideCustomView();


    boolean isVideoState();

}

File: scaffold/src/main/java/top/xuqingquan/web/publics/IVideo.kt
package top.xuqingquan.web.publics

import android.view.View
import android.webkit.WebChromeClient

interface IVideo {

    fun onShowCustomView(view: View, callback: WebChromeClient.CustomViewCallback)

    fun onHideCustomView()

    fun isVideoState(): Boolean

}
